tests/garage/torch/modules/test_categorical_gru_module.py

```diff
@@ -0,0 +1,33 @@
+"""Test CategoricalGRUModule."""
+import pickle
+
+import numpy as np
+import pytest
+import torch
+from torch.distributions import Categorical
+import torch.nn as nn
+
+from garage.torch.modules.categorical_gru_module import CategoricalGRUModule
+
+
+class TestCategoricalGRUModule:
+    def setup_method(self):
+        self.batch_size = 1
+        self.time_step = 1
+        self.feature_shape = 2
+        self.output_dim = 1
+        self.dtype = torch.float32
+
+        self.input = torch.full(
+            (self.batch_size, self.time_step, self.feature_shape), 1.)
+    
+    def test_dist(self):
+        model = CategoricalGRUModule(input_dim=self.feature_shape, output_dim=self.output_dim, hidden_dim=1)
+        dist = model(self.input)
+        assert isinstance(dist, Categorical)
+    
+    @pytest.mark.parametrize('output_dim', [1, 2, 5, 10])
+    def test_output_normalized(self, output_dim):
+        model = CategoricalGRUModule(input_dim=self.feature_shape, output_dim=output_dim, hidden_dim=1)
+        dist = model(self.input)
+        return np.isclose(dist.probs.squeeze().sum().detach().numpy(), 1)
```

Comment:
Will this test throw any exceptions if it's those values aren't close? 

You likely need to change this to an assert statement instead.

Reply:
good catch!